.idea/libraries/php_openapi.xml

```diff
@@ -0,0 +1,10 @@
+<component name="libraryTable">
+  <library name="php-openapi">
+    <CLASSES>
+      <root url="jar://$USER_HOME$/Applications/PhpStorm EAP.app/Contents/plugins/php/lib/php-openapi.jar!/" />
+      <root url="jar://$USER_HOME$/Applications/PhpStorm EAP.app/Contents/plugins/php/lib/php.jar!/" />
```

Comment:
This is the only problematic part. Other paths are relative, but this presumes OS X, EAP, and installation to `~/Applications` (which is not even default).

Maybe there could be multiple files for Win, OS X, …?